### ISSUE TITLE
Rename 'icebox' milestone to 'future'?

### ISSUE THREAD

**Issue by jespercockx** [agda/agda#3396]
I feel bad for assigning the `icebox` label to issues because it seems to imply that the issue has been abandoned or we don't think it is worth it. What it actually means is just that we don't have concrete plans to implement the new feature for the next release. Hence I propose we rename `icebox` to `future`, which would also nicely mirror the name of the branch.

**Comment by UlfNorell** [comment]
In that case we should have both `icebox` and `future`. There are certainly some issues that are in the icebox for exactly the implied reason.

What's the problem with just bumping the milestone for the issues you feel bad about? Do we really need a level between `icebox` and `2.6.1`?

**Comment by jespercockx** [comment]
I feel there is a difference between 

1. "we have concrete plans to implement this for the next release"
2. "this would be nice to have but currently no-one is working on it"
3. "this issue has been abandoned / we don't think it's worth it". 

Personally I think issues that fall under 3. should be closed with an 'abandoned' or 'wontfix' label, so we don't need a separate milestone for them. For 2. I think 'future' would be a more appropriate name than 'icebox'.

**Comment by UlfNorell** [comment]
Yeah, maybe. We might want to do a pass over the icebox and close some issues in this case, to avoid giving false hope.

**Comment by asr** [comment]
Please document the final decision in `HACKING.md` (see, for example, the 'Closing issues' section).

We should not have closed issues with `icebox` milestone. A closed issue should has a `X.Y.Z` milestone or has no milestone at all.

**Comment by nad** [comment]
> 1. "we have concrete plans to implement this for the next release"
> 2. "this would be nice to have but currently no-one is working on it"
> 3. "this issue has been abandoned / we don't think it's worth it". 

You also have variations like "none of the main developers think this is a priority, but contributions would be welcome". Perhaps we don't need separate labels for everything, we could indicate what is meant in a comment. However, things that we definitely do not intend to implement even if we get a pull request should be closed.

**Comment by andreasabel** [comment]
We could have milestones like this (say master at 2.6.0, latest release 2.5.4.2):
* `2.5.4.3` : critical issues, demanding a bugfix release asap
* `2.6.0` : features planned for next release
* `future` : features the core developers want to work on, but not for the next release
* `PR-welcome` : valuable features the core developers will probably not get to work on
* `debatable` : features whose value is unclear

These are basically 5 priorities.  Everything below that would be closed as `wontfix`.
